src/home_audit.ts

```diff
@@ -145,6 +145,11 @@ export function validate_address({ assessment_type, dwelling_unit_type, ...addre
     } else if(!dwelling_unit_type_enum.includes(dwelling_unit_type)) {
         address_errors.blocker['dwelling_unit_type'] = `Dwelling unit must one of '${dwelling_unit_type_enum.join("', '")}'`
     }
+
+    // Check if address 2 is required
+    if (dwelling_unit_type && dwelling_unit_type=='apartment_unit' && !address['address2']) {
```

Comment:
Change `apartment_unit` to get from dwelling_unit_type_enum

Reply:
dwelling_unit_type_enum is wrongly named i.e. is actaully list let me see if I can find enum class